Content.Server/GameObjects/Components/Damage/DestructibleComponent.cs

```diff
@@ -56,9 +98,10 @@ protected override void DestructionBehavior()
                 var pos = Owner.Transform.Coordinates;
                 ActSystem.HandleDestruction(Owner,
                     true); //This will call IDestroyAct.OnDestroy on this component (and all other components on this entity)
-                if (DestroySound != string.Empty)
+                if (!Owner.Deleted && DestroySounds.Count > 0)
                 {
-                    EntitySystem.Get<AudioSystem>().PlayAtCoords(DestroySound, pos);
+                    EntitySystem.Get<AudioSystem>()
+                        .PlayAtCoords(DestroySounds.Count == 1 ? DestroySounds[0] : _random.Pick(DestroySounds), pos);
```

Comment:
You can use a SoundCollection instead, also maybe use AudioHelpers to give this a random pitch? (0.125)